### PR TITLE
Notes and readme about mixins and functions

### PR DESCRIPTION
**Pull request by hadasfa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ import "monday-ui-style/dist/index.min.css";
 import IconName from "monday-ui-style/src/Icons/IconName.svg";
 ```
 
+**Mixins and functions**: We export multiple scss mixins and function helpers that can be used in your application if you use SCSS. All mixins can be imported as demonstrated below.
```
Comment (hadasfa) [inline README.md:34]:
```suggestion
**Mixins and functions**: We export multiple scss mixins and function helpers that can be used in your application if you use SASS. All mixins can be imported as demonstrated below.
```

File: scripts/copy-public-folder.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# compilation converts .scss files to pure css files (which does not support any sass abilities.)
+# This is why we must export the actual scss files (without a compilation).
+# This script's purpose is to export a folder containing public sass helpers by copying them to the dist folder.
+
+# Define the source and destination paths
+source_folder="./src/$1"
+source_index="./src/_$1.scss"
+destination_folder="./dist/$1"
+destination_index="./dist/_$1.scss"
+# Copy the mixins folder and its contents
```
Comment (YossiSaadi) [inline scripts/copy-public-folder.sh:13]: maybe omit the word mixins as it refers to both functions and mixins

File: scripts/copy-public-folder.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+cp -r "$source_folder" "$destination_folder"
+
+# Copy the mixins index as partial
```
Comment (YossiSaadi) [inline scripts/copy-public-folder.sh:16]: maybe omit the word mixins as it refers to both functions and mixins

**Review by YossiSaadi — APPROVED** [review]

**Review by SergeyRoyt — APPROVED** [review]

File: scripts/copy-public-folder.sh (hunk 2 above)
Comment (hadasfa) [inline scripts/copy-public-folder.sh:13]:
```suggestion
# Copy the public folder and its contents
```

File: scripts/copy-public-folder.sh (hunk 3 above)
Comment (hadasfa) [inline scripts/copy-public-folder.sh:16]:
```suggestion
# Copy the public folder index file as partial
```

**Pull request merged**